<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Камера</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,4">
            <item>
             <widget class="QPushButton" name="chooseCameraButton">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Выбрать камеру</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="disconnectCameraButton">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Отключить камеру</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Видео</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,4">
            <item>
             <widget class="QPushButton" name="uploadVideoButton">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Загрузить видео</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearVideoMonitorButton">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Очистить экран</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Изображение</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,4">
            <item>
             <widget class="QPushButton" name="uploadImageButton">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Загрузить изображение</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearImageMonitorButton">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Очистить экран</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
